### PR TITLE
Updated some comments around planning area actions and tests

### PR DESCRIPTION
**Pull request by runforyourlives**

### CODE REVIEW

File: src/planscape/planning/views.py [hunk 1]
```diff
@@ -252,7 +254,10 @@ def list_planning_areas(request: HttpRequest) -> HttpResponse:
     Retrieves all planning areas for a user.
     Requires a logged in user.  Users can see only their owned planning_areas.
 
-    Returns: A list of planning areas in JSON form.
+    Returns: A list of planning areas in JSON form.  Each planning area JSON will also include
+        two metadata fields:
+      scenario_count: number of scenarios for the planning area returned.
+      scenario_latest_updated_at: latest datetime across all scenarios or None if there are no scenarios.
```
Comment (sachiyuan) [inline src/planscape/planning/views.py:260]: Can we add what specific format the time stamp will be stored in?

Comment (runforyourlives) [reply src/planscape/planning/views.py]: Done.  (It's, well, datetime.)

Comment (sachiyuan) [reply src/planscape/planning/views.py]: Oh lol, sorry my brain just read "datetime" as "timestamp" for some reason. Thanks for adding the example!

**Review by sachiyuan — APPROVED** [review]
Lgtm

**Pull request merged**